src/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -241,13 +238,9 @@ private static int[] ComputeTokens(
         var lastStartCharacter = 0;
 
         var tokenTypeMap = SemanticTokensSchema.GetSchema(supportsVisualStudioExtensions).TokenTypeMap;
```

Comment:
If you use a FixedSizeArrayBuilder, you can just use .Add

Reply:
That type is so nice.